webapp/src/Controller/Jury/RejudgingController.php

```diff
@@ -37,10 +37,10 @@
 use Symfony\Component\PropertyAccess\PropertyAccess;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 
 /**
  * @Route("/jury/rejudgings")
- * @IsGranted("ROLE_JURY")
```

Comment:
Is this with intent?

Reply:
I think this is an old commit, right? It should be back again.

Reply:
Its outdated, so probably you're right.